presto-main/src/test/java/io/prestosql/operator/scalar/TestStringFunctions.java

```diff
@@ -239,19 +240,19 @@ public void testReplace()
         assertFunction("REPLACE('::\uD801\uDC2D::', ':', '')", createVarcharType(5), "\uD801\uDC2D"); //\uD801\uDC2D is one character
         assertFunction("REPLACE('\u00D6sterreich', '\u00D6', 'Oe')", createVarcharType(32), "Oesterreich");
 
-        assertFunction("CAST(REPLACE(utf8(from_hex('CE')), '', 'X') AS VARBINARY)", VARBINARY, new SqlVarbinary(new byte[] {'X', (byte) 0xCE, 'X'}));
+        assertFunction("CAST(REPLACE(utf8(from_hex('CE')), '', 'X') AS VARBINARY)", VARBINARY, sqlVarbinary('X', 0xCE, 'X'));
 
         assertFunction("CAST(REPLACE('abc' || utf8(from_hex('CE')), '', 'X') AS VARBINARY)",
                 VARBINARY,
-                new SqlVarbinary(new byte[] {'X', 'a', 'X', 'b', 'X', 'c', 'X', (byte) 0xCE, 'X'}));
+                sqlVarbinary('X', 'a', 'X', 'b', 'X', 'c', 'X', 0xCE, 'X'));
```

Comment:
This could be more tersely declared as:
```
sqlVarbinaryFromIso88591("XaXbXcX\u00CEX")
```

(technique used internally by protobufs)